vtexdocs/RapiDoc
# Fix/empty section

#### What is the purpose of this pull request?

Remove the title of the Auth section if it is empty and removing input margin in the server section (Base URL) if there are no inputs.

#### What problem is this solving?

When the sections Header Auth and Base URL are empty (without inputs) the section title and input margin still appear, making the page look as if there was an error and de inputs were not rendered correctly.

#### How should this be manually tested?

Open the [preview](https://deploy-preview-148--elated-hoover-5c29bf.netlify.app/), then open the _Generate authentication token_ API reference page and check if the section Header Auth is hidden and the Base URL section doesn't have a empty space after the header.

#### Screenshots or example usage

![Captura de tela de 2023-01-04 09-31-24](https://user-images.githubusercontent.com/62757720/210558236-952c6503-f0bf-4c8c-97ad-ab064ac16454.png)

#### Types of changes
